### PR TITLE
Add class Exception and added the ability to boot from composer

### PR DESCRIPTION
**Pull request by sergey144010**
Added checking feature availability imap_open. If php.ini is off extension = php_imap.dll, then when you run the class is silent (@imap_open?). Where to find the error? Therefore added the exception class ImapClientException. Added namespace. Added the ability to download through the composer. Added part of the Install in the README.md.

### CODE REVIEW

File: example-direct.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+var_dump($emails);
+
+// add new folder for archive
+$imap->addFolder('archive');
+
+// move the first email to archive
+$imap->moveMessage($emails[0]['uid'], 'archive');
+
+// delete second message
+$imap->deleteMessage($emails[1]['uid']);
```
Comment (islammohamed) [inline example-direct.php]: new line please

**Comment by SSilence** [comment]
can you fix the conflicts? then I will merge this

**Comment by sergey144010** [comment]
Ok. Need a little time, a little busy now.

**Comment by SSilence** [comment]
Thanks a lot!!

**Pull request merged**